react-native/react/tracker/action-render.desktop.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+/* @flow */
+
+import React from '../base-react'
+import BaseComponent from '../base-component'
+
+export default class ActionRender extends BaseComponent {
+  constructor (props) {
```

Comment:
I'm happy to do a project wide removal of these, feel free to merge in and leave it to me.